tfs/src/main/java/hudson/plugins/tfs/TeamPullRequestMergedDetailsAction.java

```diff
@@ -24,7 +24,7 @@
     private static final long serialVersionUID = 1L;
     private static final String URL_NAME = "team-pullRequestMergedDetails";
 
-    public GitPullRequestEx gitPullRequest;
+    public transient GitPullRequestEx gitPullRequest;

```

Comment:
🐛 It is going to break the event persistence on the disk, because the field won't be stored anymore.
And this action is persisted from what I see